azure-mongodb.yml

```diff
@@ -111,6 +111,15 @@ provision:
     type: string
     details: Name for your shard key
     default: uniqueKey
+  - field_name: indexes
+    required: false
+    type: string
+    details: A comma-separated list of non-unique indexes.
```

Comment:
This could have been an array rather than a comma-separated list like [AWS DynamoDB Table](https://github.com/cloudfoundry/csb-brokerpak-aws/blob/afe92bf52be957ff388ac8fc514f4e7fb73bef28/aws-dynamodb-table.yml#L67). I don't have a strong feeling about what's best. Maybe DynamoDB was a mistake and a comma-separated list is better. Maybe we should check in with the team on this to make sure we have consensus.

Reply:
We discussed and decided that we are happy with the implemented approach.